### PR TITLE
Implements login via Github

### LINKED ISSUE #135: Add Github authentication to UI
https://developer.github.com/v3/oauth/

### PR DESCRIPTION
**Pull request by solkaz**
Fixes #135 

User is able to log in and out with a Github account after they have authorized it. This PR relies on https://github.com/maryvilledev/skilldirectory/pull/211; without it, the front end will not be able to get the access token for the user

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,2 @@
 REACT_APP_API=http://localhost:8080
+REACT_APP_GITHUB_CLIENT_ID=eafec0dcd096e688249d
```
Comment (dane-johnson) [inline .env]: Is this an id for the app or the user?

File: src/LoginLink.jsx [hunk 2]
```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 import cookie from 'react-cookie';
-import { browserHistory, Link } from 'react-router';
+import { browserHistory } from 'react-router';
+
+const client_id = (process.env.REACT_APP_GITHUB_CLIENT_ID);
+const authURL = `https://github.com/login/oauth/authorize?scope=user:email%20read:org&client_id=${client_id}`;
```
Comment (dane-johnson) [inline src/LoginLink.jsx:6]: Maybe this should be an environment variable?

File: .env (hunk 1 above)
Comment (thebho) [reply .env]: That is the default for localhost, i'll need to create another Client_ID and secret for dev under the new sandbox.maryvilledevcenter.io

File: src/LoginLink.jsx (hunk 2 above)
Comment (solkaz) [reply src/LoginLink.jsx:6]: We'll be hitting the same endpoint regardless of which environment is being used, so I don't think that's needed

**Review by thebho — APPROVED** [review]

**Pull request merged**